doc/source/tune/user-guide.rst

```diff
@@ -588,6 +588,8 @@ By default, syncing occurs every 300 seconds. To change the frequency of syncing
 
 Note that uploading only happens when global experiment state is collected, and the frequency of this is determined by the ``TUNE_GLOBAL_CHECKPOINT_S`` environment variable. So the true upload period is given by ``max(TUNE_CLOUD_SYNC_S, TUNE_GLOBAL_CHECKPOINT_S)``.
 
+Make sure that worker nodes have the write access to the cloud storage. Failing to do so would cause error messages like ``Error message (1): fatal error: Unable to locate credentials``.
+For AWS set up, this involves adding an IamInstanceProfile configuration for worker nodes.
```

Comment:
Nice! Can you also, as part of this PR, include the information on this issue on the docs:

https://github.com/ray-project/ray/issues/9327

Specifically, I think it should go into https://docs.ray.io/en/latest/cluster/aws-tips.html page, and then we can link this here.